### PR TITLE
`FDiv` and function calls in error propagation mode

### PR DESCRIPTION
**Pull request by Brant-Skywalker**

### CODE REVIEW

File: enzyme/test/Integration/ForwardError/err_empty.c [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+// RUN: if [ %llvmver -ge 12 ]; then %clang -std=c11 -g -O1 %s -S -emit-llvm -o -  %newLoadClangEnzyme -Xclang -verify; fi
+// RUN: if [ %llvmver -ge 12 ]; then %clang -std=c11 -g -O2 %s -S -emit-llvm -o -  %newLoadClangEnzyme -Xclang -verify; fi
+// RUN: if [ %llvmver -ge 12 ]; then %clang -std=c11 -g -O3 %s -S -emit-llvm -o -  %newLoadClangEnzyme -Xclang -verify; fi
+
+extern double __enzyme_error_estimate(void*, double, double);
+
+double unknown(double in);
+
+double g(double in) {
+    return unknown(in); // expected-error {{Enzyme: No forward mode derivative found for unknown}}
```
Comment (wsmoses) [inline enzyme/test/Integration/ForwardError/err_empty.c:15]: could we change the error message to "no forward mode error function found" or something?

Comment (Brant-Skywalker) [reply enzyme/test/Integration/ForwardError/err_empty.c:15]: I believe it's hardcoded here: https://github.com/EnzymeAD/Enzyme/blob/main/enzyme/Enzyme/EnzymeLogic.cpp#L4745

Comment (wsmoses) [reply enzyme/test/Integration/ForwardError/err_empty.c:15]: Ah yes, maybe we could do an if statement there where if mode == ForwardError ... else ...

Comment (Brant-Skywalker) [reply enzyme/test/Integration/ForwardError/err_empty.c]: > Ah yes, maybe we could do an if statement there where if mode == ForwardError ... else ...

Ok done

**Review by wsmoses — APPROVED** [review]

**Pull request merged**